src/components/Toast/ErrorToast.tsx

```diff
@@ -0,0 +1,39 @@
+import Link from "next/link"
+import { FC } from "react"
+import { toast } from "react-hot-toast"
+
+import { Toast, TransactionToastComponentProps } from "@/components/Toast"
+
+import { FortIconWarning } from "@/icons"
+
+export const ErrorToast: FC<TransactionToastComponentProps> = ({
+  message,
+  txHash,
+  ...toastProps
+}) => {
+  return (
+    <Toast
+      className="flex items-center bg-red-500 text-white"
+      onDismiss={() => toast.dismiss(txHash)}
+      {...toastProps}
+    >
+      <div className="inline-flex h-8 w-8 flex-shrink-0 items-center justify-center rounded-lg bg-red-800">
```

Comment:
I feel like this would look better without the background on it, but its a matter of aesthetic opinion.